framework/web/AssetManager.php

```diff
@@ -211,8 +211,6 @@ public function init()
         $this->basePath = Yii::getAlias($this->basePath);
         if (!is_dir($this->basePath)) {
```

Comment:
What about this check? Shouldn't it be moved too? It seems like there is no reason to check `basePath` if nothing will be published there

Reply:
May be it should be moved, but the issue is about checking $basePath for writability only.

Also I've found one problem in test.
To emulate read only dir I run chmod($path, 0555) in the test. But after that developer can have a problem to switch to another branch. 
Can you suggest something? Thanks.

Reply:
You can create this folder in `setUp` and delete it in `tearDown` method

Reply:
Or you can `chmod` it back in `tearDown`

Reply:
Done, thanks.